### DISCUSSION TITLE
Animated GIF support?

### DISCUSSION THREAD

**Discussion by 2jfw** [airsdk/Adobe-Runtime-Support#2556] [Q&A]
@ajwfrost: Is there any chance to get support for animated GIFs in a future release of AIR?

**Comment by itlancer** [comment]
By the way. Right now you can use https://github.com/theturtle32/Flash-Animated-GIF-Library but it have a lot of issues with some GIF variants.

**Reply by 2jfw** [reply]
Thanks for pointing out!
Just came across this one here (https://github.com/Devwarlt/as3-gif-support) but did not check neither of them so far

**Reply by ajwfrost** [reply]
This is the sort of thing we'd want to enable via some extension mechanism (i.e. additional APIs/objects for use with ANEs) - mostly everything would be in place for this anyway I think (via BitmapData access?). Animated gifs would make a good case study for it :-)
